ui/StatusQ/src/StatusQ/Core/LocaleUtils.qml

```diff
@@ -162,6 +171,34 @@ QtObject {
         return value.toLocaleString(loc, formatString)
     }
 
+    // TODO use JS Intl.RelativeTimeFormat in Qt 6?
```

Comment:
Not sure if it's already there, if not it would be nice to create an issue to keep track of this TODO.